cli/src/config/templates.toml

```diff
@@ -14,6 +14,12 @@ if(remote,
 
 commit_summary = 'format_commit_summary_with_refs(self, branches)'
 
+annotate_commit_summary = '''
+self.change_id().shortest(8) ++ " " ++ 
+self.commit_id().shortest(8) ++ " " ++ 
+self.author().name() ++ " " ++ format_timestamp(self.author().timestamp())
+'''
```

Comment:
nits: 
- use `format_short_id()` and `format_short_signature()`
- use `committer.timestamp()` for consistency with log templates
- signature field will need some padding, but we don't have padding function right now https://github.com/martinvonz/jj/issues/3183

```
separate(" ",
  format_short_id(change_id),
  format_short_id(commit_id),
  format_short_signature(author),
  format_timestamp(committer.timestamp()),
)
```

Reply:
done